src/python_testing/TestChoiceConformanceSupport.py

```diff
@@ -19,11 +19,11 @@
 import xml.etree.ElementTree as ElementTree
 
 import jinja2
-from choice_conformance_support import (evaluate_attribute_choice_conformance, evaluate_command_choice_conformance,
```

Comment:
the underlying tests should be moved in `matter_testing_infrastructure` and made run as the `tests =` for pw_python_package.

https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/matter_testing_infrastructure/BUILD.gn#L35


However if we want to do this later that is fine as well, feel free to close this comment and do as a followup. The change would be:
   - move these files and add to tests = in build.gn
   - remove them from tests.yaml (as they start running as part of `check`)
 
 At that point we can also remove them from exceptions from the globbing implementation of the runner.